server/src/main/java/org/spine3/server/aggregate/Aggregate.java

```diff
@@ -293,30 +317,60 @@ private void apply(Iterable<? extends Message> eventMessages, CommandContext com
         }
     }
 
-    private void apply(Message eventOrMsg, CommandContext commandContext)
+    /**
+     * Applies the passed event message or {@code Event} to the aggregate.
+     *
+     * @param eventOrMessage an event message or {@code Event}
+     * @param commandContext the context of the command which handler generated the message or event
+     * @throws InvocationTargetException if the applier method throws the exception
+     * @see #ensureEventMessage(Message)
+     */
+    private void apply(Message eventOrMessage, CommandContext commandContext)
```

Comment:
This is so to support import of events. It does look strange, but it should be OK for a private method.